llama/sync.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-# Run in the llama directory
-
 set -e
 
+# Run in the llama directory
+cd $(dirname ${BASH_SOURCE[0]})
```

Comment:
I don't think this is supported on non-BASH shells. But in general, I'd prefer to separate out these syncing changes to a separate patch (and thing else that is helpful but not related to the core of this patch).